src/components/organisms/deployments/tableContent.tsx

```diff
@@ -198,16 +198,17 @@ export const DeploymentsTableContent = ({
 
 				<TBody>
 					{sortedDeployments.map(({ buildId, createdAt, deploymentId, sessionStats, state }) => (
-						<Tr className="hover:bg-gray-1300" key={deploymentId}>
-							<Td
-								className="w-1/8 cursor-pointer pl-4"
-								onClick={() => goToDeploymentSessions(deploymentId)}
-							>
+						<Tr
+							className="hover:bg-gray-1300"
+							key={deploymentId}
+							onClick={() => goToDeploymentSessions(deploymentId)}
+						>
+							<Td className="w-1/8 cursor-pointer pl-4">
```

Comment:
All `Td`s should be clickable and take you to the deployment sessions without the filters.